packages/@okta/i18n/src/properties/login.properties

```diff
@@ -800,8 +800,8 @@ oie.okta_verify.push.send = Send push notification
 oie.okta_verify.signed_nonce.label = Use Okta Verify on this device
 
 ## Google Authenticator
-oie.google_authenticator.totp.title = Google Authenticator
-oie.google_authenticator.totp.enterCodeText = Enter your Google Authenticator passcode
+oie.google_authenticator.otp.title = Google Authenticator
+oie.google_authenticator.otp.enterCodeText = Enter your Google Authenticator passcode
```

Comment:
change the key name to, want to avoid camel case
`oie.google_authenticator.otp.entercode.text`

Reply:
@pradeepdewda-okta  why is that ? Did we make a decision on it ? There are many string with camelCase here in this file I guess.

Reply:
@magizh-okta No decision. My approach is if we touch the code then we can fix it along the way, won't cause any harm.